printer_mods/hartk1213/Voron2.4_Trident_Pins_Mod/README.md

```diff
@@ -34,6 +35,31 @@ here is what the XY Joint will look like
 ![XY Joint](Images/1.jpg)
 ![XY Joint](Images/2.jpg)
 
+There is a new file for the A Drive to go along with [Arkeets MGN12](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/arkeet/mgn12) mod that has the X endstop on the toolhead 
```

Comment:
Can you make this a relative link? just makes it less likely to break if something changes.

Reply:
just removed it and replaced it with 2.4r2/trident carriage